cmd/zpool/zpool_main.c

```diff
@@ -2389,6 +2392,9 @@ zpool_do_import(int argc, char **argv)
 				mntopts = optarg;
 			}
 			break;
+		case 'p':
+			flags |= ZFS_IMPORT_SCRUB_PAUSE;
+			break;
```

Comment:
For pool recovery, importing readonly would be a better option.

I get that "import + cancel/pause scrub" could be useful in a few specific scenarios, but from a user interface design it seems arbitrary.  There are a lot of other specific scenarios where other combo options could be helpful.  e.g. what if I'm resilvering and for some reason the hardware doesn't like that?  What if I'm doing a background destroy?  What if one of the disks is sort of busted and I want to do a "import + offline <dev>"?